### DISCUSSION TITLE
How to Serialize the screen model and save to xml file

### DISCUSSION THREAD

**Discussion by kengjilu** [canton7/Stylet#345] [Q&A] [answered]
I try to serialze my screen model, and there's  an error happened.Then I try to Add  "[XmlIgnore]" before those object like below:DisplayName, ScreenState, ScreenState,IsActive,Parent,View. The problem was solved.

**Comment by canton7** [comment, marked as answer]
ViewModels aren't meant to be serialized. You should write a model which is supposed to be serialized, and create one of those when you want to generate XML.
